api/test/integration/test_security_GET_endpoints.tavern.yaml

```diff
@@ -428,6 +502,62 @@ stages:
         data:
           total_affected_items: 0
 
+  # GET /security/policies
+  - name: Try to show the default policies only
+    request:
+      <<: *all_policies_request
+      params:
+        resource_type: default
+    response:
+      status_code: 200
+      json:
+        error: !anyint
+        data:
+          affected_items:
+            - id: !anyint
+              name: !anystr
+              policy: !anything
+              resource_type: default
+          failed_items: []
+          total_affected_items: !anyint
+          total_failed_items: 0
+
+  # GET /security/policies
+  - name: Try to show the protected policies only when there is no protected policies available
+    request:
+      <<: *all_policies_request
+      params:
+        resource_type: protected
+    response:
+      status_code: 200
+      json:
+        error: !anyint
+        data:
+          affected_items: []
```

Comment:
Consider adding a testing `protected` resource to test this.

Reply:
The reason no `protected` were added was to ensure the function does not return anything when specifying a resource type not present in the database. We are testing no `default` or `user` security resources are being returned instead.